spec/models/profile_spec.rb

```diff
@@ -575,21 +922,59 @@
     end
   end
 
+  # TODO: does deactivating make sense for a non-active profile? Should we prevent it?
+  # TODO: related: should we test against an active profile here?
```

Comment:
We used to activate and immediately deactivate. I think Amir fixed that, but the method names still include "deactivate."